doc/input-output-reference/src/overview/group-hybrid-model.tex

```diff
@@ -1,43 +1,192 @@
-\section{Group - Hybrid Model}\label{group---hybrid-model}
+\section{Group -- Hybrid Model}\label{group-hybrid-model}
 
-This object is to provide a hybrid model feature to enhance the accuracy of the building energy simulation for existing buildings. The hybrid model feature replaces internal thermal mass and infiltration air flow rates with zone air temperature data. The hybrid model feature calculates a temperature capacitance multiplier, equivalent to the zone internal thermal mass and infiltration air change per hour (ACH) based on the inverse modeling of the heat balance algorithms. 
+This object provides a hybrid model feature to enhance energy simulations for existing buildings by inversely solving hard-to-measure model input parameters. More specifically, zone internal thermal mass, air infiltration rate (ACH), and people count are usually hard to be measured directly yet important in building energy simulations. The hybrid model takes measured zone air temperature, humidity ratio, or CO$_2$ concentration as inputs and inversely solve those unknown parameters based on the zone air heat, moisture, or CO$_2$ balance algorithms. The inversely solved internal thermal mass, air infiltration rate, and people count can then be used to substitute the original simplified assumptions to enhance the energy simulation.
 
-\subsubsection{Inputs}\label{inputs-hm}
+\subsubsection{HybridModel:Zone}\label{inputs-hm}
 
 \paragraph{Field: Name}\label{field-name-hm}
```

Comment:
All occurrences of \paragraph should be followed by a blank line so the text below starts a new paragraph.